packages/backend/src/open_payments/payment/outgoing/service.test.ts

```diff
@@ -290,6 +294,22 @@ describe('OutgoingPaymentService', (): void => {
     receiver = incomingPayment.getUrl(receiverWalletAddress)
 
     amtDelivered = BigInt(0)
+
+    // Make receivers non-local by default
+    receiverGet = receiverService.get
+    jest
+      .spyOn(receiverService, 'get')
+      .mockImplementation(async (url: string) => {
+        // call original instead of receiverService.get to avoid infinite loop
+        const receiver = await receiverGet.call(receiverService, url)
+        if (receiver) {
+          // "as any" to circumvent "readonly" check (compile time only) to allow overriding "isLocal" here
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          ;(receiver.isLocal as any) = false
+          return receiver
+        }
+        return undefined
+      })
```

Comment:
There were all sorts of problems with the unit tests (quote/outgoing payemnt) after implementing the logic to pay via local or ilp methods. Before, all methods were implicitly ILP. But now all the existing tests are recognized as local (since there is only 1 rafiki instance in the tests). Took some mocking here to make them all remote by default (and mocking is fine - receiver is not what we're testing here).